### ISSUE TITLE
One and the other construction in copula: singular subjects, plural copula, more than one subject?

### ISSUE THREAD

**Issue by Stormur** [UniversalDependencies/docs#714]
I don't know if the following problem has already surfaced, but I was not able to locate it here or in the guidelines, as it seems to me of a mixed nature, and I don't actually know how to aptly name it; I don't think it to be an instance of clefting.

The starting point is this Latin sentence (which I am slightly modifying and simplifying in its components to highlight its structure) which perplexed as to how it is best annotated:

_libros                           habent,  qui                               libri                                sunt    alter_                                
`book.MASC-PL-ACC   have      that.MASC-PL-NOM   book.MASC-PL-NOM    are    other.MASC-SING-NOM`

_bonus                                   et           alter                                        malus_
`good.MASC-SING-NOM      and        other.MASC-SING-NOM        bad.MASC-SING-NOM`

i.e., quite literally, 'they have (some) books, which are books one good and the other bad', which I don't know how grammatical it is in English, but something very similar can happen at least in Italian, too.

Now, my first consideration is that there are too many elements. It seems we migh split it into two sentences:

_qui libri sunt: alter (est) bonus et alter (est) malus_ 
'that [they] are books: one is good and the other is bad'

Zero copula is allowed in Latin. One sure thing should be that _qui_ is (part of) `nsubj`, and _sunt_ `cop`, probably depending on _libri_, but how to join the other pieces? `parataxis` does not look as a good choice, as this isn't just a juxtaposition. 

1) By the way, the problem would be solved if we had a relative pronoun as a connector, e.g. _quorum_ 'of which [masculine plural genitive]':

_libros habent, qui libri sunt quorum alter bonus et alter malus_ 'they have some books, which are books of which one is good and the other is bad'

Here, I see this analysis to be quite unproblematic:

```
1	Libros	_	NOUN	_	_	2	obj	_	_
2	habent	_	VERB	_	_	0	root	_	_
3	,	_	PUNCT	_	_	5	punct	_	_
4	qui	_	PRON	_	_	5	nsubj	_	_
5	libri	_	NOUN	_	_	1	acl:relcl	_	_
6	sunt	_	AUX	_	_	5	cop	_	_
7	quorum	_	PRON	_	_	8	nmod	_	_
8	alter	_	DET	_	_	9	nsubj	_	_
9	bonus	_	ADJ	_	_	5	acl:relcl	_	_
10	et	_	CCONJ	_	_	12	cc	_	_
11	alter	_	DET	_	_	12	nsubj	_	_
12	malus	_	ADJ	_	_	9	conj	_	_
13	.	_	PUNCT	_	_	2	punct	_	_
```

That is, we have the two copulae connected by means of an `acl:relcl` relation, each with its own subject: _libri_ and _alter_/_alter_.

2) But without, I am not so sure about keeping `acl`, which does not seem justified by anything.

```
1	Libros	_	NOUN	_	_	2	obj	_	_
2	habent	_	VERB	_	_	0	root	_	_
3	,	_	PUNCT	_	_	5	punct	_	_
4	qui	_	PRON	_	_	5	nsubj	_	_
5	libri	_	NOUN	_	_	1	acl:relcl	_	_
6	sunt	_	AUX	_	_	5	cop	_	_
7	alter	_	DET	_	_	8	nsubj	_	_
8	bonus	_	ADJ	_	_	5	?	_	_
9	et	_	CCONJ	_	_	11	cc	_	_
10	alter	_	DET	_	_	11	nsubj	_	_
11	malus	_	ADJ	_	_	8	conj	_	_
12	.	_	PUNCT	_	_	2	punct	_	_
```

I was thinking rather of a very awkward `ccomp`, in vein with the treatment of copulae with a clause as their "nominal" part. One comparative consideration that contributed to this was that, at least in **very** colloquial Italian (not every speaker would agree on its grammaticality), it is possible to formulate this sentence with the _universal_ complementiser _che_ 'that' (not clear if it should be seen as a relative element): _questi libri sono libri che uno è bello e l'altro brutto_ lit. 'these books are books that one is good and the other bad'. `xcomp` has to be excluded, since subjects can be different.

3) Another solution still I have thought of:

```
1	Libros	_	NOUN	_	_	2	obj	_	_
2	habent	_	VERB	_	_	0	root	_	_
3	,	_	PUNCT	_	_	5	punct	_	_
4	qui	_	DET	_	PronType=Rel	5	det	_	_
5	libri	_	NOUN	_	_	8	nsubj	_	_
6	sunt	_	AUX	_	_	8	cop	_	_
7	alter	_	DET	_	_	8	xcomp	_	_
8	bonus	_	ADJ	_	_	1	acl:relcl	_	_
9	et	_	CCONJ	_	_	11	cc	_	_
10	alter	_	DET	_	_	11	xcomp	_	_
11	malus	_	ADJ	_	_	8	conj	_	_
12	.	_	PUNCT	_	_	2	punct	_	_
```

That is, now the interpretation is: 'they have books, which books are one good and the other bad'. _qui_ would act here as a relative determiner: the major problem is that this syntax of _qui_ (the repetition of the antecedent) is very rare, even if attested. But then: would be the `xcomp` for the _alter_'s sensible? 

---

So, out of the three possibilities, my _formal_ preference goes to the 3), as it is much cleaner, even if it presents a quite unusual syntax of _qui_ (as it blurs the line between its use as an indefinite determiner and a relative pronoun), but I am not sure it represents the right interpretation.

Has anybody come across similar cases, or are there some indications on how to treat them buried in the guidelines? Thanks for any help and sorry for the lengthy post!

---

Still another variant, more similar to 3) : 

 ```
1	Libros	_	NOUN	_	_	2	obj	_	_
2	habent	_	VERB	_	_	0	root	_	_
3	,	_	PUNCT	_	_	5	punct	_	_
4	qui	_	DET	_	PronType=Rel	5	det	_	_
5	libri	_	NOUN	_	_	7	nsubj	_	_
6	sunt	_	AUX	_	_	7	cop	_	_
7	alter	_	DET	_	_	1	acl:relcl	_	_
8	bonus	_	ADJ	_	_	7	amod	_	_
9	et	_	CCONJ	_	_	10	cc	_	_
10	alter	_	DET	_	_	7	conj	_	_
11	malus	_	ADJ	_	_	10	amod	_	_
12	.	_	PUNCT	_	_	2	punct	_	_
```

Here the _alter_'s are seen as the predications, modified by different `amod`s in the two conjuncts. The repeated element is the head, and this allows to preserve symmetry even if the adjective is replaced with a more complex element, like `obl`s _domi_ 'in the house' / _ruri_ 'in the countryside' (i.e. 'two books, one at home and the other in the countryside')

**Comment by sylvainkahane** [comment]
French has similar constructions. I  think that two unusual constructions are involved.

First, the  possibility to have the noun with the relative pronoun (the pronoun _lequel_ (plural _lesquels_) is very formal, in both examples):

```
_J'ai des livres, lesquels sont en bon état._
I have books, which are in good shape.

_J'ai des livres, lesquels livres sont en bon état._
I have books, which books are in good shape.
```

Second, the coordination of predicative adjectives with determiners:

```
_Certains livres sont abimés._
Some books are damaged.

_D'autres livres sont en bon état._
Other books are in good shape.

=> _Les livres sont certains abimés et d'autres en bon état._
The  books are some damaged et other in good shape.
```

Both constructions can be combined:

```
_J'ai des livres, lesquels livres sont certains abimés et d'autres en bon état._
I have book, which books are some damaged and other in good shape.
```

Both construction are complex and I'm not sure how to annotate them. Your solution 3 seems the best one, except for the determiner _alter_, which looks like a floating quantifier (#581). I'm not sure to know what solution has been adopted, but not `xcomp`. Probably `advmod`.

**Comment by Stormur** [comment]
Hm, about the solution with `advmod`, I have to say that I would find it undesirable, _sic stantibus rebus_ (as things stand :smiley:) in UD, since it would mean to annotate _alter_ as an `ADV`, with all consequences regarding its morphological traits and its concordance with _liber_, here. This actually ties in into a bigger problem wortthy of its own issue, namely to allow `advmod` deprel for non-`ADV`, but I am digressing.

**Comment by Stormur** [comment]
Regarding the discussion about floating quantifiers, as I have just commented there, I think that the right interpretation is `xcomp`; I feel that `advmod` would distort the syntax of the sentence on too many levels.

I have thought that these constructions might be probably simply seen as an instance of nominal ellipsis: what is left out is a repetition of the noun that already appears as a subject.

Taking your French example: we can see that we have both _certains **livres** abimés_ 'some damaged books' and  _autres **livres** en bon état_ 'other books in a good state'. So, instead of saying: 

_J'ai des livres, lesquels livres sont certains livres abimés et d'autres (des?) livres en bon état_
'I have some books, which books are some damaged books and other books in good shape'

, we just omit the _livres_ following the first one. 

If this is case, we then just promote according to the [hierarchy](https://universaldependencies.org/u/overview/specific-syntax.html#ellipsis-in-nominals) `amod > nummod > det > nmod > case`. keeping the same dependencies. So (hoping to get everything right in French):

```
det(livres,lequels) , nsubj(abimés,livres)
cop(abimés,sont) , det(abimés,certains)
conj(abimés,autres) , nmod(autres,en bon état)
```

What is important is that both noun phrases stays co-ordinated and keep their internal relations. 

By the way, might this be a case to move `nmod` up in the hierarchy?

**Comment by sylvainkahane** [comment]
Does _alter_ work as a pronoun? Because for French, both _certains_ 'some' and _d'autres_ 'other(s)' work both as DET and PRON and in this construction they are pronouns. It is impossible to have what you propose (*_J'ai des livres, lesquels livres sont certains livres abimés et d'autres livres en bon état._) with a noun _livres_  after _certains_ or _d'autres_. Exactly as in floating quatifiers :

```
_Mes livres sont tous abimés._
My  books are all damaged.
_*Mes livres sont tous les livres abimés._
```
Floating quantifiers are modifiers and they don't involve ellipsis. In SUD, I will just say that there are `mod`: [examples](http://match.grew.fr/?corpus=SUD_French-GSD@latest&custom=5ef4d8f634db4&clustering=DEP.lemma). This `mod` relation is automatically converted in `obl:mod` in UD ([examples](http://match.grew.fr/?corpus=UD_French-GSD@2.6&custom=5ef4dab98b1f5&clustering=DEP.lemma)), but we never really discussed this choice, because we don't know what is UD's solution.

**Comment by Stormur** [comment]
> Does _alter_ work as a pronoun? Because for French, both _certains_ 'some' and _d'autres_ 'other(s)' work both as DET and PRON and in this construction they are pronouns. It is impossible to have what you propose (*_J'ai des livres, lesquels livres sont certains livres abimés et d'autres livres en bon état._) with a noun _livres_ after _certains_ or _d'autres_. Exactly as in floating quatifiers :
> 
> ```
> _Mes livres sont tous abimés._
> My  books are all damaged.
> _*Mes livres sont tous les livres abimés._
> ```
> 

Am I wrong or can you generally say _certains livres (abimés)_ 'some (damaged) books'? In any case, with _alter_ 'other' in Latin you can, and it is a determiner. But then, every determiner `DET`, as a nominal, can stand alone and act as a "pronoun", however it always stays a `DET` (the same way that an `ADJ` used without a nominal head does not become a `PRON` or a `NOUN`). You might say:

_Hic liber est alter liber bonus_ 'this book is another good book'

But an interpretation of _alter_ as a "floating quantifier", or rather as a secondary predication, meaning something like 'this book, as the "other" of more than one book, is good', might still stand. _alter_ might be replaced by _totus_ 'every/all': 'the book, in its entirety, is good'. Maybe copulae have some restrictions regarding secondary predications, but one could also replace _alter_ with _apertus_ 'opened' and mean: 'this book, when it is open/considered as opened/..., is good', in contrast with 'the opened book is good', and so on. In Latin every secondary predication can have such ambiguity. The limit seems to be semantic/pragmatic (for example, absence/presence of contrast).

However, "italianizing" your example, this ambiguity vanishes, as we can have:
* _I miei libri sono tutti (libri) danneggiati_ 'My books are all damaged (books)' - no article, "floating quantifier" - _libri_ 'books' can be omitted.
* _I miei libri sono tutti [i libri/quelli] danneggiati_ 'My books are all the damaged ones/books' - with article OR pronominal form of _quei_ 'those (pl. masc.)' - _tutti_ 'all (pl. masc.)' determines the phrase. The repetition of _libri_ is pedantic, but still sounds well.

Now, with _certi_ 'some (pl. masc.)', which should be the equivalent of fr. _certains_, it functions indeed differently. One difference is that _certi_ never admits the article as _tutti_, so there is only one probable reading:
* _I miei libri sono certi (libri) danneggiati..._ 'My books are some damaged ones/(books)...' - leaving out _libri_ sounds awkward and very conditioned by the context ('some damaged things'...), at the limit of what is acceptable. But:
* _i miei libri sono certi danneggiati e certi intatti_ 'My books are some damaged and some untouched' - Here the contrast makes the sentence good and at the same time clear that it is used as a "floating quantifier". And also:
* _i miei libri sono certi libri danneggiati e certi libri intatti_ is OK, the repetition gives emphasis.There is again ambiguity: _libri_ might be "assigned" to _certi_ or to the adjective: 'some books damaged and some books untouched' vs. 'some (are) damaged books and some (are) untouched books'. Also _certi_ as a determiner of the whole phrase might be possible, but very awkward, sounding somehow cryptic (which books are alluded to?).

---

Now, what I think this all says? That some interpretations are preferred over others and that a language might or might not present ways to disambiguate between them (in the Italian case with _certi_ ambiguity is given by the absence of the article, as in Latin). Latin does not. So, in this case _alter_ might work either ways, but in the end I think the preferred interpretation is that of a "floating quantifier", or secondary predications, as it is the only one creating the contrast (also etymologically, _alter_ has an innate contrastive meaning). Contrast is symmetric, while nominal ellipsis is not (at least in UD).

Also, from a semantic point of view, _libros habent, qui libri sunt alter bonus et alter malus_ and _libros habent, qui libri sunt bonus et malus_ are nearly identical, the former highlighting contrast and the latter not.

**Comment by dan-zeman** [comment]
I think _bonus/malus_ is something like a [secondary predication](https://universaldependencies.org/u/overview/complex-syntax.html#secondary-predicates) about _libri_. Probably optional depictive, following the terminology laid out in the guidelines. That would make them `acl` of _libri_, and @Stormur's first option would apply, only I don't think they are necessarily `acl:relcl`:

<pre>
1	Libros	_	NOUN	_	_	2	obj	_	_
2	habent	_	VERB	_	_	0	root	_	_
3	,	_	PUNCT	_	_	5	punct	_	_
4	qui	_	PRON	_	_	5	nsubj	_	_
5	libri	_	NOUN	_	_	1	acl:relcl	_	_
6	sunt	_	AUX	_	_	5	cop	_	_
7	quorum	_	PRON	_	_	8	nmod	_	_
8	alter	_	DET	_	_	9	nsubj	_	_
9	bonus	_	ADJ	_	_	5	acl	_	_
10	et	_	CCONJ	_	_	12	cc	_	_
11	alter	_	DET	_	_	12	nsubj	_	_
12	malus	_	ADJ	_	_	9	conj	_	_
13	.	_	PUNCT	_	_	2	punct	_	_
</pre>

There are two points that make this example different from the usual cases of optional depictives though. First, the superordinate clause is copular and the modified nominal is its predicate rather than argument. And second, the "secondary" predicate gets its own subject, _alter_. Still I think I prefer this analysis. One could say that the three clauses with nonverbal predicates are at the same level as conjuncts or parataxis members (all would be relative clauses modifying _libros_) but that does not make much sense because one would not utter it without the _bonus/malus_ part: _?Libros habent, qui libri sunt._ I also don't like the `xcomp` solution (@Stormur's no. 3) because I think _alter_ represents the subject of _bonus_ rather than another predication about an invisible subject of _bonus_.

**Comment by Stormur** [comment]
So, if I understood well, you would prefer the analysis where the _alter bonus/alter malus_ part depends on _libri_ by means of a non-relative (because there would be no relative element _quorum_) `acl`, assuming a zero-copula there?

So:

<pre>
1	Libros	_	NOUN	_	_	2	obj	_	_
2	habent	_	VERB	_	_	0	root	_	_
3	,	_	PUNCT	_	_	5	punct	_	_
4	qui	_	PRON	_	_	5	nsubj	_	_
5	libri	_	NOUN	_	_	1	acl:relcl	_	_
6	sunt	_	AUX	_	_	5	cop	_	_
7	alter	_	DET	_	_	9	nsubj	_	_
8	bonus	_	ADJ	_	_	5	acl	_	_
9	et	_	CCONJ	_	_	12	cc	_	_
10	alter	_	DET	_	_	12	nsubj	_	_
11	malus	_	ADJ	_	_	9	conj	_	_
12	.	_	PUNCT	_	_	2	punct	_	_
</pre>

Anyway, in solution no. 3 the subject of _alter_ would be visibile in _libri_, and the mismatch in `Number` given by the split of a plural subject over two different predications.

The problem now is if the treatment above should be different from another sentence with nearly identical meaning and very similar structure:

_Libros habent, qui sunt alter bonus, alter malus, alter magnus, alter parvus, alter..._

This is what led me to consider _alter_ as a depictive, rather than _bonus_. Here it looks like `cop(bonus,sunt)` is the predicate with _qui_ as subject, and all the others are linked by means of `conj`: `conj(bonus,malus) conj(bonus,magnus) conj(bonus,parvus) conj(bonus,...)`  So I thought it should be possible to "fit" _libri_ into this structure without radically altering it (i.e., _libri_ replacing _bonus_ as the predicate and shifting everything else to `acl`). Or, in such a case, should we assume that the predicate of _sunt_ is itself a clause (_alter bonus..._)?

**Comment by dan-zeman** [comment]
I tried to find a similar case in Czech. I found one, and I don't completely agree with the analysis that my conversion script produced there (http://hdl.handle.net/11346/PMLTQ-RAAY). The equivalent of _alter bonus_ is attached as an `appos` of the equivalent of _libri_. We shouldn't do that in UD because we restrict apposition to nominals (unlike in the Prague-style annotation, where it is occasionally used with clauses). But maybe it is not so bad to use `conj` instead in UD after all?

`conj(libri, bonus)`; `conj(bonus, malus)`; `nsubj(bonus, alter)` ...

**Comment by Stormur** [comment]
I have come again upon this construction, and I think that the solution might be quite simpler than thought. I also think that the copular construction is the most peculiar one of this kind, and this might have misled us. 

So, the basic observation is that we have a block consisting of two or more, usually elliptical, co-ordinated clauses that has to be "hooked" to the main clause in some way. I would distinguish two main cases:
1. this block "expands" an element that is already present in the main clause, or
2. the first clause of the block coincides with the main clause.

So, based on the examples I used:
* lat. _Habeo duos libros, alterum domi et alterum ruri_ / it. _Ho due libri, uno a casa e l'altro in campagna_ 'I have two books, one at home and the other in the country', we have the first case;
* lat. _Illi libri sunt alter domi et alter ruri_ / it. _Quei libri sono uno a casa e l'altro in campagna_ 'Those books are one at home and the other in the country', we have the second case.

1) 
The first case is simple: it is not dissimilar from an _id est clause_ of explanatory nature, which is nothing else than a particular type of co-ordination and for which we are using the subtyped relation `conj:expl`. So I would annotate:

`conj:expl(habeo,alterum) orphan(alterum,domi) conj(domi,ruri)`

as this is like saying _I have two books; I have one at home and I have the other in the country_. And the same goes for other sentences like _They went in (two) different directions, Ann eastwards and Bob westwards_, and also _These are the books, one good and one bad_: `conj:expl(books,good) nsubj(good,one) conj(good,bad)`.
In the case of an attached copular clause, I think the best bet would be to simply keep `conj:expl` for symmetry (instead e.g. of `acl`) and to link it to the corrisponding noun (nominal clause <-> nominal element): _I have two books, one good and one bad_ -> `conj:expl(books,good) nsubj(good,one) conj(good,bad)` (here I have to admit that I don't know how Latin would treat this with cases and thus underlying structures: _alterum bonum et alterum malum_ or _alter bonus et alter malus_, or...? Is it allowed?).

2)
The second case is trickier because of the role of the "contrasting" elements (_one_/_other_/...), but still follows the same logic. 

(a) _They went one eastwards and one westwards_

has the same basic structure as (b) _They went eastwards and westwards_ (ambiguous co-ordination) and/or (c) _One went eastwards and the other westwards_ (no "common" subject expressed): it is a fusion of the two models, and so it happens that the subjects of version (a) become (are "demoted" to) "secondary subjects" in the resulting sentence, i.e. secondary predicates. One fact I think is interesting is that it seems to me that there are some preferential choices for such secondary elements which might depend from some notion of definiteness, compare for example _They went Ann eastwards and Bob westwards_ to _Ann and Bob went one eastwards and one westwards_... but the discussion becomes too complicated here.

In any case, I would annotate (a) as 

```
1	They	_	PRON	_	_	2	nsubj	_	_
2	went	_	VERB	_	_	0	root	_	_
3	one	_	PRON	_	_	2	advcl:pred?	_	_
4	eastwards	_	ADV	_	_	2	advmod	_	_
5	and	_	CCONJ	_	_	7	cc	_	_
6	one	_	PRON	_	_	7	(advcl:pred?)/orphan	_	_
7	westwards	_	ADV	_	_	2	conj	_	_
```

Now, the best relation for _one_ is probably `advcl`, as discussed in #476, maybe with a subtype for this special use.  
If we remove the `advcl:pred`s from (a)'s annotation, we get the annotation for (b); if we remove the `nsubj` and "promote" the `advcl:pred`s from secondary to "primary subjects", we get the annotation for (c).

And so, going back to the example (with copula) that prompted the whole discussion (Dante, _De Monarchia_):

_[...] relationes, quae relationes sunt altera sub ambitu paternitatis et altera sub ambitu dominationis_
"officially" translated as 'relationships [...] which are respectively relationships of 'paternity' and of 'lordship''

I would annotate:

```
12	relationes	relatio	NOUN	_	_7	obl	_	_
[...]
21	quae	qui	PRON	_	PronType=Rel	22	det	_	_
22	relationes	relatio	NOUN	_	_	26	nsubj	_	_
23	sunt	sum	AUX	_	_	26	cop	_	_
24	altera	alter	DET	_	_	26	advcl:pred	_	_
25	sub	sub	ADP	_	_	26	case	_	_
26	ambitu	ambitus	NOUN	_	_	12	acl:relcl	_	_
27	paternitatis	paternitas	NOUN	_	_	26	nmod	_	_
28	et	et	CCONJ	_	_	31	cc	_	_
29	altera	alter	DET	_	_	31	advcl:pred	_	_
30	sub	sub	ADP	_	_	31	case	_	_
31	ambitu	ambitus	NOUN	_	_	26	conj	_	_
32	dominationis	dominatio	NOUN	_	_	31	nmod	_	_
```

The subject _quae relationes_ is meant to be common to both symmetrical co-ordinated clauses. As before, we have the related sentences _quae relationes sunt sub ambitu paternitatis et sub ambitu dominationis_ and _Altera (relatio) [est] sub ambitu paternitatis et altera (relatio) [est] sub ambitu dominationis_.

----

So, to sum it up (sorry again for the lengthy post), I think that these kinds of constructions which confused us so in the end all boil down to co-ordinations and secondary predications, so `conj`(`:expl`) and `advcl`(`:pred`).
